src/components/Person.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+    <p className="Person__partner">
+      {isMarried
+        ? `My ${sex === 'f' ? 'husbands' : 'wife'} name is ${partnerName}`
```

Comment:
Will be better to create separate variable before return statement in component, save here final married status message and use this variable in jsx
Such way helps to avoid adding inner code logic in jsx for better understanding final html